assets/facets.js

```diff
@@ -7,7 +7,14 @@ class FacetFiltersForm extends HTMLElement {
       this.onSubmitHandler(event);
     }, 500);
 
-    this.querySelector('form').addEventListener('input', this.debouncedOnSubmit.bind(this));
+    const facetForm = this.querySelector('form');
+    facetForm.addEventListener('input', this.debouncedOnSubmit.bind(this));
+    facetForm.addEventListener('keydown', (event) => {
```

Comment:
Do we still need this?

Reply:
With it: https://screenshot.click/25-55-fenh4-yw0jf.mp4

Without it: https://screenshot.click/25-54-a52ez-frkgr.mp4

That is because it doesn't listen to the `input ` event listener

Reply:
It originally was checking the input but not updating the products 🤔 retested and it works now probably. Maybe it was caching in the video above. Removed 🤔